src/cuttlefish_datatypes.erl

```diff
@@ -193,6 +193,16 @@ to_string(Value, MaybeExtendedDatatype) ->
             {error, {type, {Value, MaybeExtendedDatatype}}}
     end.
 
+-spec to_quoted_string(term(), datatype()) -> string() | cuttlefish_error:error().
+to_quoted_string(V, Type) when Type =:= ip; Type =:= string;
+                               element(1, Type) =:= enum; element(1, Type) =:= duration;
+                               Type =:= bytesize; Type =:= file; Type =:= directory ->
+  quote(to_string(V, Type));
+to_quoted_string(V, Type) ->
+  to_string(V, Type).
+
+quote(Text) when is_list(Text) -> "\"" ++ Text ++ "\"".
```

Comment:
maybe not an issue, but there is perhaps a need for quote escape here?
(Not sure if we use config generation at all)